.php-cs-fixer.dist.php

```diff
@@ -118,7 +117,9 @@
         ],
         'logical_operators' => true,
         'native_constant_invocation' => true,
-        'native_function_invocation' => true,
+        'native_function_invocation' => [
+            'include' => ['@internal'],
```

Comment:
Keep old behavior as it has been changed. More details https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/v3.0.0/UPGRADE-v3.md#changed-default-values-of-options